src/ugene_version.pri

```diff
@@ -14,7 +14,7 @@ U2_DISTRIBUTION_INFO=sources
 # int version levels for executables
 UGENE_VER_MAJOR=38
 UGENE_VER_MINOR=0
-UGENE_VER_SUFFIX=-dev
+UGENE_VER_SUFFIX=
```

Comment:
Why do you want to delete this? We have to hate it in "master"

Reply:
I used the wrong target branch.